app/controllers/application_controller.rb

```diff
@@ -52,6 +44,6 @@ def set_locale
   end
 
   def store_request_id
-    RequestStore.store[:request_id] = request.uuid
+    Instrumentation.append_to_log(request_id: request.uuid)
```

Comment:
Not sure about the naming for the use case of `request_id`, we do 2 things with the `request_id`: include it in the logs and store it to use it in the api client. This name implies that only does logging so it wont be obvious from where the client gets the `request_id`.

I can't think of a good alternative right now though...